### PR TITLE
feat: single relations against wasi branches

### PR DESCRIPTION
**Pull request by dbcfd**

### CODE REVIEW

File: src/api.rs [hunk 1]
```diff
@@ -25,7 +25,7 @@ pub struct BlockData<T: Serialize> {
     /// Header for block
     pub header: BlockHeader,
     /// Data for block
-    #[serde(skip_serializing_if = "Option::is_none")]
+    //#[serde(skip_serializing_if = "Option::is_none")]
```
Comment (dav1do) [inline src/api.rs:28]: Was this change intentional?

File: Cargo.toml [hunk 2]
```diff
@@ -13,20 +13,23 @@ publish = false
 
 [dependencies]
 anyhow = "1"
-ceramic-event = { git = "https://github.com/3box/rust-ceramic", branch = "main" }
-json-patch = { version = "1.0.0", features = ["diff"] }
+#ceramic-event = { git = "https://github.com/ceramicnetwork/rust-ceramic", branch = "main" }
```
Comment (dav1do) [inline Cargo.toml:16]: Maybe delete the unused dependencies? Or I guess we could merge the wasi changes and go back to main.

File: src/lib.rs [hunk 3]
```diff
@@ -146,12 +147,12 @@ impl<S: Signer> CeramicHttpClient<S> {
...
         &self,
         model_id: &StreamId,
-    ) -> anyhow::Result<api::CreateRequest<DagCborEncoded>> {
+    ) -> anyhow::Result<api::CreateRequest<DeterministicInitEvent>> {
         if !model_id.is_model() {
             anyhow::bail!("StreamId was not a model");
         }
         let args = EventArgs::new_with_parent(&self.signer, model_id);
-        let commit = args.init()?;
+        let _commit = args.init()?;
```
Comment (dav1do) [inline src/lib.rs:155]: Was this wrong before? I don't really understand this change. Why do we need to do this if we don't use it? Is it to make sure it's valid or something?

**Review by dav1do — COMMENTED** [review]
The code changes look fine but I don't totally understand things. Should we get the `feat/wasi` branch going again so we can clean up the toml a bit here?

File: src/api.rs (hunk 1 above)
Comment (dbcfd) [reply src/api.rs:28]: Yes, that looks to be one of the pieces that is required for deterministic/single events.

File: Cargo.toml (hunk 2 above)
Comment (dbcfd) [reply Cargo.toml:16]: Kind of would like to just merge the wasi changes. Otherwise I can delete.

File: src/lib.rs (hunk 3 above)
Comment (dbcfd) [reply src/lib.rs:155]: Yep, it's just a validity check. We don't actually use that information when making the deterministic init commit.

File: src/api.rs (hunk 1 above)
Comment (dav1do) [reply src/api.rs:28]: got it, maybe nice to delete but all good

File: Cargo.toml (hunk 2 above)
Comment (dav1do) [reply Cargo.toml:16]: I don't have objections, it looks like that one needs a rebase and format but should be fine. 

Was curious if we have any policy around forks (3box, ceramicnetwork, personal account).. obviously it'd be nicer to not have them, and we don't want to give the impression we're maintaining something when we just want a fix

**Review by dav1do — APPROVED** [review]
